packages/tests/integration/schema/internalComment.graphql

```diff
@@ -0,0 +1,26 @@
+extend type Mutation {
+  saveInternalComment(input: SaveInternalCommentInput!): SaveInternalCommentResult!
+}
+
+type InternalComment {
```

Comment:
Suggestion: Fwiw since we don't really need it, can you delete all of the GraphQL scaffolding from this new entity? I've only recently started removing it (unnecessary GQL scaffolding in the tests), but it just ends up being boilerplate that we don't really need for Joist's integration tests.